.github/workflows/pull_requests.yml

```diff
@@ -41,14 +53,21 @@ jobs:
           key: cache-gradle-${{ hashFiles('gradle/wrapper/gradle-wrapper.properties') }}
           restore-keys: |
             cache-gradle-
+      - name: Delete Maven Settings file
+        run: rm ~/.m2/settings.xml
```

Comment:
After installing  java 17
i faced a problem with the maven publish plugin `net.linguica.maven-settings`

it seems that we have some servers here `~/.m2/settings.xml` but we don’t have `~/.m2/settings-security.xml` (with master password) to encrypt the servers passwords

The gradle was crashing here
https://github.com/mark-vieira/gradle-maven-settings-plugin/blob/master/src/main/groovy/net/linguica/gradle/maven/settings/LocalMavenSettingsLoader.java#L89

My workaround was to just delete the `~/.m2/settings.xml` before triggering the build :smile:

Reply:
it might be because of `setup-java` github action which i am using to install java 17
https://github.com/actions/setup-java/tree/main#maven-options

Reply:
it should be fine for now i think as long as it doesn’t impact development or final publishing